xml/ha_storage_protection.xml

```diff
@@ -332,6 +332,15 @@
         is considered delivered. The timeout should be long enough for the node to
         detect that it needs to self-fence.
        </para>
+       <para>
+        However, if the <literal>msgwait</literal> timeout is too long,
+        a fenced cluster node might rejoin before the fencing action returns.
+        This can be mitigated by setting the <varname>SBD_DELAY_START</varname>
+        parameter in the SBD configuration, as described in
+        <xref linkend="pro.ha.storage.protect.sbd.config"/>
+        of
```

Comment:
I guess it should be the other way round: first the step and then the procedure?